src/control/lib/netdetect/netdetect.go

```diff
@@ -288,6 +289,15 @@ func Init(parent context.Context) (context.Context, error) {
 	}
 	ndc.numNUMANodes = numNUMANodes(ndc.topology)
 	ndc.numaAware = ndc.numNUMANodes > 0
+	if ndc.numaAware {
+		cores, err := getCoreCount(ndc.topology)
+		if err != nil {
+			return nil, err
+		}
+		ndc.coresPerNuma = cores / ndc.numNUMANodes
```

Comment:
I think this is a fine simplifying assumption, but I wonder if it will always be true. I could imagine some weird hardware with unbalanced sockets (mixture of on-die CPUs and GPUs or something like that). Probably not something we're likely to encounter on storage nodes, though.

Reply:
Agreed, this assumption won't hold for architectures with asymmetric CPU design across sockets.